modules/twinklewarn.js

```diff
@@ -1171,6 +1177,17 @@ Twinkle.warn.callback.change_category = function twinklewarnCallbackChangeCatego
 			// create the options
 			createEntries(groupContents, optgroup, false);
 		});
+	} else { // All of the above
+		['level1', 'level2', 'level3', 'level4', 'level4im'].forEach(function(lvl) {
+			value = lvl;
+			Object.keys(Twinkle.warn.messages.levels).forEach(function(groupLabel) {
+				createEntries(Twinkle.warn.messages.levels[groupLabel], sub_group, true);
+			});
```

Comment:
Odd style. 
```js
$.each(Twinkle.warn.messages.levels, function(_, group) {
    createEntries(group, sub_group, true);
});
```
  
?

Reply:
Dunno about odd, at least no more odd than js vs jQuery.  I had it that way at first but changed it since I didn't like the empty variable in `$.each`.  I sort of like the visual difference from the level 1-4 section, but do agree this isn't particularly Twinkley.

Reply:
Actually it's better to use `Object.values` here (since we're only using the value) like I did [back there](https://github.com/azatoth/twinkle/pull/747#discussion_r359567205) wehre you laughed at me :)

Where both key and value are needed, I think `$.each` is the best way. The next best way would be to use [Object.entries() along with array destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries#Iterating_through_an_Object) but bah, that's ES6.

Reply:
Changed.

>Where both key and value are needed, I think $.each is the best way

Bien sûr!